### PR TITLE
Use shared methods from set protocol utils

### PR DESCRIPTION
**Pull request by asoong**
You can merge your stuff in @bweick and i'll work around you. Just make sure to rebase for your next PR.

### CODE REVIEW

File: test/core/extensions/coreIssuance.spec.ts [hunk 1]
```diff
@@ -162,7 +162,7 @@ contract('CoreIssuance', accounts => {
 
     describe('when the set was not created through core', async () => {
       beforeEach(async () => {
-        subjectSetToIssue = NULL_ADDRESS;
+        subjectSetToIssue = SetProtocolUtils.CONSTANTS.NULL_ADDRESS;
```
Comment (felix2feng) [inline test/core/extensions/coreIssuance.spec.ts]: Maybe we can declare
```
const { NULL_ADDRESS }` =  SetProtocolUtils.CONSTANTS;
```
above, so we only need to reference NULL_ADDRESS in the tests themselves

File: test/core/extensions/coreIssuanceOrder.spec.ts [hunk 2]
```diff
@@ -317,9 +317,9 @@ contract('CoreIssuanceOrder', accounts => {
 
         // Standard 0x order without fees, see zeroExExchangeWrapper.spec.ts for clarity on body
         const zeroExOrder: ZeroExOrder = Utils.generateZeroExOrder(
-          NULL_ADDRESS,                       // senderAddress
+          Utils.CONSTANTS.NULL_ADDRESS,       // senderAddress
```
Comment (felix2feng) [inline test/core/extensions/coreIssuanceOrder.spec.ts]: Same here re: declaration on top

File: test/core/extensions/coreIssuance.spec.ts (hunk 1 above)
Comment (asoong) [reply test/core/extensions/coreIssuance.spec.ts]: +1

File: test/core/extensions/coreIssuanceOrder.spec.ts (hunk 2 above)
Comment (asoong) [reply test/core/extensions/coreIssuanceOrder.spec.ts]: +1

**Pull request merged**